src/main/java/org/jabref/gui/documentviewer/DocumentViewerController.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) 2003-2016 JabRef contributors.
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
```

Comment:
Good catch! Some of the code was written in an era, when such copyright notices were still included 😄